### PR TITLE
Fix for a performance issue with writers initialization

### PR DESCRIPTION
**Pull request by sergiorg-hpc**

### CODE REVIEW

File: src/reports/library/implementation_interface.hpp [hunk 1]
```diff
@@ -103,7 +103,7 @@ struct ParallelImplementation {
             mpi_size_type = MPI_UINT32_T;
         }
 
-        int global_rank, global_size;
+        int global_rank, global_size, local_size;
```
Comment (sergiorg-hpc) [inline src/reports/library/implementation_interface.hpp:106]: Not strictly needed, but better to change it to `local_size` instead.

**Review by jorblancoa — APPROVED** [review]

**Comment by pramodk** [comment]
Are we tagging new version and deploying this on BB5?

**Comment by jorblancoa** [comment]
We should do that yes.
@sergiorg-hpc could you create a spack PR with a new version in libsonata-report?
Just replace existing one for 0.1.3

**Comment by sergiorg-hpc** [comment]
> We should do that yes.
> @sergiorg-hpc could you create a spack PR with a new version in libsonata-report?
> Just replace existing one for 0.1.3

Perfect, here's the PR for that:
https://github.com/BlueBrain/spack/pull/945

**Pull request merged**